src/ansys/fluent/core/workflow.py

```diff
@@ -476,33 +538,66 @@ def set_state(self, args: dict) -> None:
         Parameters
         ----------
         args : dict
-            New argument state.
+            New argument's state..
```

Comment:
I'm not convinced that you are checking the suggestions(?) @PipKat is doing a great job but she has declared - which is entirely reasonable - that she cannot know what is a suitable description of every item.

Reply:
Sorry for this issue. Corrected it now.